src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,45 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    public static final Integer STORE_SIZE = 10;
+    private Object[] keyArr;
+    private Object[] valueArr;
+    private int lengthOfActualValue = 0;
```

Comment:
* parametrize the arrays
* Storage is not always a store :)
* do not use abbreviations or cut words
* class fields of primitive types have default values
* wrappers as class fields can be useful in some cases, but this is not one of them
overall try to pay more attention to the naming
```suggestion
    public static final int MAX_SIZE = 10;
    private K[] keys;
    private V[] values;
    private int size;
```